src/main/java/org/jenkinsci/plugins/workflow/log/ConsoleAnnotators.java

```diff
@@ -69,7 +71,7 @@
                 @SuppressWarnings("deprecation") // TODO still used in the AnnotatedLargeText version
                 Cipher sym = PASSING_ANNOTATOR.decrypt();
                 try (ObjectInputStream ois = new ObjectInputStreamEx(new GZIPInputStream(
-                        new CipherInputStream(new ByteArrayInputStream(Base64.decode(base64.toCharArray())), sym)),
+                        new CipherInputStream(new ByteArrayInputStream(Base64.getDecoder().decode(base64)), sym)),
```

Comment:
Fine, using ISO-8859-1.

Reply:
```suggestion
                        new CipherInputStream(new ByteArrayInputStream(Base64.getDecoder().decode(base64.getBytes(StandardCharsets.UTF_8))), sym)),
```

Reply:
The version in this PR was actually preferable to my mind, but yes I think the overriding concern is minimizing the diff to the official version.